### PR TITLE
Improve Voting Feature

### PR DESCRIPTION
**Pull request by NaiRobley**
#### What does this PR do?
Improve voting feature by solving UI/UX issues #15 and #16 

#### Screenshots
View of a user who up-voted a data request:
image

View of a user who down-voted a data request:
image

View of a user who has not voted or is not logged in:
image

View of votes and comment on the listing of data requests:
image

### CODE REVIEW

File: ckanext/datarequests/controllers/ui_controller.py [hunk 1]
```diff
@@ -21,6 +21,8 @@
 
 import ckan.lib.base as base
 import ckan.model as model
+from ckan.common import config
+import ckan.lib.mailer as mailer
```
Comment (DavidLemayian) [inline ckanext/datarequests/controllers/ui_controller.py]: Interesting.. were these missing before?

**Comment by DavidLemayian** [comment]
30853136-9ca6af20-a2b6-11e7-9cd4-46f620aa1212

File: ckanext/datarequests/controllers/ui_controller.py (hunk 1 above)
Comment (NaiRobley) [reply ckanext/datarequests/controllers/ui_controller.py]: No. I had rebased on our Andela fork and this is part of the work a teammate was doing. I forgot to remove it while fixing the conflicts, but I will.

**Comment by NaiRobley** [comment]
Working on the rating position.

**Comment by NaiRobley** [comment]
@DavidLemayian adjusted the rating position as requested. Updated screenshots on PR

**Pull request merged**